src/command_modules/azure-cli-appservice/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 (unreleased)
 ++++++++++++++++++++
 * webapp: add reliability fixes in configuring source control (#3245)
+* webapp: remove --node-version from 'webapp config update', as server has abandoned this setting long while ago
```

Comment:
I prefer to mention it in help, as that can be referred easier by users. Releasing notes is a onetime read thing